capstone/src/main/java/com/google/sps/data/Room.java

```diff
@@ -0,0 +1,92 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.Entity;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.sps.service.DatabaseService;
+
+public class Room {
+    public static final String ROOM_ENTITY_NAME = "Room";
+    public static final String TITLE_PROPERTY_KEY = "title";
+    public static final String DESCRIPTION_PROPERTY_KEY = "description";
+    public static final String HOST_PROPERTY_KEY = "host";
+    public static final String FOLLOWERS_PROPERTY_KEY = "followers";
+    public static final String FORMS_PROPERTY_KEY = "forms";
+    
+
+    private Entity entity;
+
+    public Room(Entity entity) {
+        this.entity = entity;
+    }
+
+    public Room(User host, String title, String description) { 
+        this.entity = new Entity(Room.ROOM_ENTITY_NAME);
+        this.entity.setProperty(Room.TITLE_PROPERTY_KEY, title);
+        this.entity.setProperty(Room.DESCRIPTION_PROPERTY_KEY, description);
+        this.entity.setProperty(Room.HOST_PROPERTY_KEY, host.getUserKey());
+        this.entity.setProperty(Room.FOLLOWERS_PROPERTY_KEY, new ArrayList<>());
```

Comment:
You don't need to set the "new ArrayList<>()" properties because datastore just recognizes them as null anyways. Instead in getter/setter methods check if those values are null and initialize them there.

Reply:
Sounds good, will fix this in this PR.